### ISSUE TITLE
[Question] file name

### ISSUE THREAD

**Issue by tamakana** [deezer/spleeter#79]
Hi. I'm not familiar to using like these things, but really like to do use this amazing tool for my music life.
How can I  get output wav files named same as track title? 
Not only "vocals.wav" , like "aaavocals.wav"  Same as track title and filename.
Is it possible?

**Comment by Faylixe** [comment]
This option is not available at the moment, we only export to a directory named after the input file name. We will consider adding a new naming scheme on the next release.

**Comment by tamakana** [comment]
Thanks for your quick response!
 I can't wait for your next release, not only about this issue but also others :)

**Comment by Faylixe** [comment]
The new version is released (v1.4.4). Please update and you can now apply the new parameters `-f` to provide an output name file format. The doc will be updated soon meanwhile here is an example to achieve what you want :

```bash
spleeter separate -f "{filename} - {instrument}.{codec}" ...
```

**Comment by tamakana** [comment]
I appreciate your support.
I've tried to use -f and got named files successfully!
But new folder wasn't created in output folder.
Should I assign any command to create new named folder as before?
When I don't use -f , named folder created to same directly as before.
I'm sorry to bother you, but I would appreciate it if you could tell me the way.
